MoreLinq.Test/TestingSequence.cs

```diff
@@ -51,59 +63,262 @@ public enum Options
     /// </summary>
     sealed class TestingSequence<T> : IEnumerable<T>, IDisposable
     {
-        bool? _disposed;
-        IEnumerable<T>? _sequence;
+        private IEnumerable<T>? _sequence;
+        private readonly Options _options;
+        private readonly int _numEnumerations;
+
```

Comment:
The reason they are separated is because the top ones have different purposes. These are the "readonly" (and `_sequence` will be readonly as well with your other comment) fields, while the ones below are the variable fields. I use blank lines to separate purpose and intent of fields.

Reply:
Thanks for the explanation. I am fine if you want to close this as resolve if the separation helps you and wish to maintain it.